Makefile

```diff
@@ -64,8 +64,8 @@ undeploy:
 	for obj in $(DEPLOY_OBJECTS) $(DEPLOY_CRDS) $(DEPLOY_CRS); do \
 		$(TEMPLATE_CMD) $$obj | kubectl delete -f - ;\
 	done	
-	 ## Delete everything for the operator from the cluster
-#	-$(TEMPLATE_CMD) $(DEPLOY_OBJECTS) $(DEPLOY_OPERATOR) $(DEPLOY_CRDS) $(DEPLOY_CRS) | kubectl delete -f -
+
+	kubectl delete scc nfd-worker
```

Comment:
Isn't this going to leave the other `$(DEPLOY_OBJECTS) $(DEPLOY_OPERATOR) $(DEPLOY_CRDS) $(DEPLOY_CRS)` objects hanging / not removed?

Reply:
This is a comment that is removed look at the resulting Makefile.